packages/flame_test/lib/src/flame_test.dart

```diff
@@ -89,9 +95,12 @@ class GameTester<T extends Game> {
     game.onGameResize(size);
 
     await game.onLoad();
+    game.onMount();
     game.update(0);
     if (game is FlameGame && makeReady) {
       await game.ready();
+      // ignore: invalid_use_of_visible_for_testing_member
```

Comment:
do we need these since this is a test file?

Reply:
For some reason the analyzer doesn't recognise it as a test.